packages/front-end/components/Features/RuleModal/RolloutFields.tsx

```diff
@@ -89,6 +94,34 @@ export default function RolloutFields({
               form.setValue("hashAttribute", v);
             }}
           />
+          <div className="mb-2">
+            <span
+              className="ml-auto link-purple cursor-pointer"
+              onClick={(e) => {
+                e.preventDefault();
+                setadvancedOptionsOpen(!advancedOptionsOpen);
+              }}
+            >
+              Advanced Options{" "}
+              {!advancedOptionsOpen ? <PiCaretDown /> : <PiCaretUp />}
+            </span>
+            {advancedOptionsOpen && (
+              <div className="mt-3">
+                <Field
+                  label="Seed"
+                  type="input"
+                  {...form.register("seed")}
```

Comment:
Under the hood do we fall back to the `rule.id` or the `feature.id`?

Reply:
I see, I thought the `id` in the `rule.seed || id` was referring to the rule, but it refers to the feature. Thanks.